.travis.yml

```diff
@@ -13,6 +16,9 @@ cache:
   - pip
 before_install:
   # - if [[ $TRAVIS_PYTHON_VERSION == 2.7 ]]; then travis_retry sudo apt-get update -qq; fi
+    - if [ "$TRAVIS_ARCH" = "ppc64le" ]; then 
+          sudo chown -Rvf $USER:$GROUP ~/.cache/pip/wheels; 
```

Comment:
any idea on why this `chown` is needed? feels like the original ppc64le is "deficient"...?